src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
```

Comment:
Link to the SCSS file is incorrect. You should link to the compiled CSS file instead. The browser cannot understand SCSS files.

Reply:
A tarefa pede para utilizar scss, que foi a aula que aprendemos, não para chamar o index.css

Reply:
sim kkk